src/test/java/org/javarosa/core/model/TriggerableDagTest.java

```diff
@@ -530,6 +530,33 @@ public void non_relevant_node_values_are_always_null_regardless_of_their_actual_
         assertThat(scenario.answerOf("/data/some-field"), is(intAnswer(42)));
     }
 
+    // Users use relevance on calculates to ensure that calculations are only run when the values
```

Comment:
Is it important to make sure that the calculation isn't executed or just that the calculation isn't "answered"? I'm guessing it's the latter (as that's what this test is about), but the comment suggests it might the former.

Reply:
Great and deep question. Now I'm no longer 100% sure of the intent. This led me to some strange docs: https://github.com/getodk/docs/pull/1547 I will come back to it.